README.md

```diff
@@ -62,29 +62,20 @@ This should create a tab in your browser with the development app that contains
 
 ### Autocompletion
 
-The autocompletion feature has 3 different modes, each requiring a different setup:
+The autocompletion feature has 2 different modes, each requiring a different setup:
```

Comment:
I'm really like this change (it makes the setup much easier) but I'm still curious... is there a place I can read more about the reason behind this?

Reply:
yeah it's because of this: https://github.com/prometheus/prometheus/pull/6872#issuecomment-703505588

Reply:
Thank you!